kbase-extension/static/kbase/js/widgets/kbaseTabs.js

```diff
@@ -1,55 +1,82 @@
 /*
-
     Easy widget to serve as a tabbed container.
 
-    var $tabs =  new kbaseTabs($('#tabs'), {
-            tabPosition : 'bottom', //or left or right or top. Defaults to 'top'
-            canDelete : true,       //whether or not the tab can be removed. Defaults to false.
-            tabs : [
-                {
-                    tab : 'T1',                                     //name of the tab
-                    content : $('<div></div>').html("I am a tab"),  //jquery object to stuff into the content
-                    canDelete : false,                              //override the canDelete param on a per tab basis
-                },
-                {
-                    tab : 'T2',
-                    content : $('<div></div>').html("I am a tab 2"),
-                },
-                {
-                    tab : 'T3',
-                    content : $('<div></div>').html("I am a tab 3"),
-                    show : true,                                    //boolean. This tab gets shown by default. If not specified, the first tab is shown
-                    showContentCallback: function // if you don't want to show the content right away, add a callback method that returns the content...
-                },
-            ],
-        }
-    );
+    It is an extension of the Bootstrap 3 tab
+    (https://getbootstrap.com/docs/3.4/javascript/#tabs).
+    
+    Below is a commented usage example:
+
+    const $tabs =  new kbaseTabs($('#tabs'), {
+        // boolean; whether a tab which does not otherwise specify this option
+        // will have a close button and can be removed. Defaults to false, overridden
+        // if set on a tab.
+        canDelete : true,
+    
+        tabs : [
+            {
+                // string; label of the tab
+                tab : 'T1',
```

Comment:
would you be able to use something that's more obviously a tab title (e.g. "Tab Title 1") - whenever I see something like `'t1'`, `'t2'`, etc., it makes me wonder if it's a variable.

Reply:
Not my original documentation, but I'll try to improve it.

Reply:
Thanks, it is much clearer now!